bundles/org.openhab.ui/web/src/components/item/item-form.vue

```diff
@@ -174,14 +257,27 @@ export default {
   },
   mounted () {
     if (!this.item) return
-    const categoryControl = this.$refs.category
-    if (!categoryControl || !categoryControl.$el) return
-    const inputElement = this.$$(categoryControl.$el).find('input')
-    this.initializeAutocomplete(inputElement)
+    if (!this.item.category) this.$set(this.item, 'category', '')
```

Comment:
Please do not do create these props here, this causes issues with dirty handling.

Either make the form work if those are not available (I think it does work without them) or set them during load time in the components using the form.
However before setting something in these components, rebase. IIRC I already pushed a fix for that, see #2408.